syntax = "proto3";
option csharp_namespace = "TomTaw.eWordIMCIS.Proto";
import "google/protobuf/any.proto";

message PageResponse {
  bool isSuccess = 1;
  string message = 2;
  PageBase pageBase=3;
  repeated google.protobuf.Any data = 4;
  int32 warnCode=5;
}

message PageBase {
  int32 currentPage=1;
  int32 pageSize=2;
  int32 totalRecords=3;
  string token = 4;
  int32 errorTimes=5;
  int32 loginStatus=6;
}